tokio/Cargo.toml

```diff
@@ -121,7 +121,7 @@ default-features = false
 optional = true
 
 [dev-dependencies]
-tokio-test = { version = "0.2.0" }
+tokio-test = { version = "0.2.0", path = "../tokio-test" }
```

Comment:
Not sure if this is kosher, but a new version of tokio-test with the `read_error` mock hasn't been released to crates.io yet.

Reply:
It *should* be ok to keep path dev-deps. What would be the reason to avoid them?

Reply:
I can't think of any reason, just wondering why it isn't already this way.